main/boost/APKBUILD

```diff
@@ -38,89 +58,94 @@ _libs="date_time
 	wserialization
 	"
 for _lib in $_libs; do
+	echo "$_options_carch" | grep -F -q -e "--without-${_lib}" || \
 	subpackages="$subpackages $pkgname-$_lib:_boostlib"
 done
+subpackages="$subpackages $pkgname-bundle:_boost_bundle:noarch"
```

Comment:
We should not need the `-bundle` subpackage. We should only need use `boost-dev` in makedepends and abuild should automatically figure out the rest.